lib/puppet/provider/ec2_instance/v2.rb

```diff
@@ -293,11 +294,28 @@ def create
         )
       end
 
+      ec2.wait_until(:instance_running, instance_ids: instance_ids)
```

Comment:
Can't we only wait when we actually want to change the value or do we have to wait in all cases?